Buildfile

```diff
@@ -8,5 +8,5 @@ cookbook.depends 'turnstile' do |turnstile|
 end
 
 profile :default do |default|
-  default.chef.run_list 'turnstile::default'
+  default.chef.run_list ['turnstile::nodejs', 'turnstile::default']
```

Comment:
Why include the Node recipe here instead of doing
```ruby
include_recipe 'turnstile::nodejs'
```
in the turnstile::default recipe?